src/interactivity/win32/Clipboard.cpp

```diff
@@ -96,7 +96,7 @@ Clipboard& Clipboard::Instance()
 // - cchData - Size of the Unicode String in characters
 // Return Value:
 // - None
-void Clipboard::StringPaste(_In_reads_(cchData) const wchar_t* const pData,
+void Clipboard::StringPaste(_In_reads_(cchData) PCWCHAR pData,

```

Comment:
in this codebase, unless we're interfacing with a Windows API we have a negative preference for the windows typedefs. :smile:

Reply:
Aah, ok, will revert this.

The declaration is little confusing though. It uses `PCWCHAR`, so I thought let's make it consistent. Maybe, we should change `PCWCHAR` to `const wchar_t* const` in the declaration too?

https://github.com/microsoft/terminal/blob/59dcbbe0e94814b94a7bfc5ae189d36c4bb0436f/src/interactivity/win32/clipboard.hpp#L33-L34